landing-page/components/sections/Partners.tsx

```diff
@@ -0,0 +1,22 @@
+import { tw, tx } from '@helpwave/common/twind'
+import Image from 'next/image'
+
+const images = {
+  'REACH': 'https://cdn.helpwave.de/partners/reach.svg',
+  'digitalHub Münsterland': 'https://cdn.helpwave.de/partners/digitalhub_muensterland.png',
+  'helpwave': 'https://cdn.helpwave.de/logo/logo.png',
+}
+
+const PartnerSection = () => {
+  const size = 128
+
+  return (
+    <div className={tw('flex gap-16 my-16 select-none overflow-x-auto justify-between items-center')}>
+      {Object.entries(images).map(([title, src]) => (
+        <Image key={title} alt={title} src={src} style={{ objectFit: 'contain' }} width={size} height={size} className={tx(`w-[${size}px] grayscale`)}/>
```

Comment:
I thought about that but as most of other website do not reference their partners by linking them. I want to avoid weird clicking behavior in that section.